index.html

```diff
@@ -157,6 +160,32 @@ <h1 class="title">About me</h1>
                             </div>
                         </div>
                     </section>
+                    <form action="https://formspree.io/f/moqzkbjd" method="post">
+
+                        <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+                        <label for="username"></label>
+                        <input type="text" id="username" name="username" placeholder="username" class="input" required maxlength="30">
+                        <br>
+                        <label for="email"></label>
+                        <input type="email" id="email" name="email" placeholder="email" class="input" required pattern=".*[.com]">
+                        <br><br>
+                        <textarea name="comment" id="comment" cols="30" rows="10" placeholder="write me something..." required maxlength="500"></textarea>
```

Comment:
- [ ] Kindly add the left margins to the input fields to match the design.

- Your design
<img width="302" alt="image" src="https://user-images.githubusercontent.com/41163571/229858505-ca00765d-c5c6-460d-9b46-e9422333cd36.png">

- Figma design
![image](https://user-images.githubusercontent.com/41163571/229572326-b555170d-2046-4106-9219-a94876fd13f4.png)